openshift/config/common/helm/babylon-config/templates/anarchy.yaml

```diff
@@ -1,10 +1,11 @@
 {{ range $anarchyNamespace := .Values.anarchy.namespaces }}
+  {{- if or ($anarchyNamespace.create) (eq ($anarchyNamespace.create | toString) "<nil>") }}
```

Comment:
Hrm... the `toString` usage seems a little odd? I did some research and saw that `default` doesn't work correctly with booleans. Maybe?
```
  {{- if or (not (hasKey $anarchyNamespace "create")) .anarchyNamespace.create }}
```

Reply:
Yes, the `toString` is kind of dirty workaround since `default` do not make difference between boolean `false` and undefined value. I've updated and tested with the `hasKey` function, it looks more clean and readable (thank you for the hint!).